### PR TITLE
Avoids HTTPS/HTTP conflicts, and linking to out-of-date documentation. 

### PR DESCRIPTION
**Pull request by DougReeder**
Avoids HTTPS/HTTP conflicts, and linking to out-of-date documentation.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Physics system for A-Frame VR, built on Cannon.js & Ammo.js",
   "main": "index.js",
   "scripts": {
-    "dev": "budo index.js:dist/aframe-physics-system.js -d examples --port 8000 -- -g browserify-shim",
+    "dev": "budo index.js:dist/aframe-physics-system.js -d examples --port 8000 --ssl -- -g browserify-shim",
```
Comment (vincentfretin) [inline package.json:7]: Please note that generating the self-signed certificate doesn't work on Ubuntu 22.04, that's one of the reason we replaced budo by webpack dev server in aframe repo. Also see #22

File: AmmoDriver.md [hunk 2]
```diff
@@ -25,12 +25,12 @@ The Ammo.js driver provides many features and new functionality that the existin
 
 ## Installation
 
-Initial installation is the same as for Cannon.js. See: [Scripts](https://github.com/donmccurdy/aframe-physics-system/blob/master/README.md#installation), then see [Including the Ammo.js Build](#including-the-ammojs-build).
+Initial installation is the same as for Cannon.js. See: [Scripts](README.md#installation), then see [Including the Ammo.js Build](#including-the-ammojs-build).
```
Comment (vincentfretin) [inline AmmoDriver.md:28]: I'm not sure if the Scripts link will work once it's deployed to ghpages. We'll need to verify this link once merged.

**Review by vincentfretin — CHANGES_REQUESTED** [review]
Thanks for the contribution.
I'm not sure why you have changes in `dist/aframe-physics-system.js` it's hard to tell. If this is just some comments, can you please remove the dist from the PR so we can just merge the documentation changes? We'll recreate the dist on next release.
For now please revert the `--ssl` option, it fails on Ubuntu 22.04. It's better to continue on the switch to webpack in the other PR #22.

File: package.json (hunk 1 above)
Comment (DougReeder) [reply package.json:7]: Ah, I wasn't aware of that. :-(

File: AmmoDriver.md (hunk 2 above)
Comment (DougReeder) [reply AmmoDriver.md:28]: The link from README.md to AmmoDriver.md works on ghpages, so this ought to as well.

**Review by vincentfretin — APPROVED** [review]
Thanks.

**Comment by DougReeder** [comment]
Links confirmed working properly on https://github.com/c-frame/aframe-physics-system/blob/master/AmmoDriver.md

**Pull request merged**